nimble/controller/src/ble_ll_dtm.c

```diff
@@ -368,8 +368,7 @@ ble_ll_dtm_tx_create_ctx(uint8_t packet_payload, uint8_t len,
     sch->sched_cb = ble_ll_dtm_tx_sched_cb;
     sch->cb_arg = &g_ble_ll_dtm_ctx;
     sch->sched_type = BLE_LL_SCHED_TYPE_DTM;
-    sch->start_time =  os_cputime_get32() +
-                                       os_cputime_usecs_to_ticks(5000);
+    sch->start_time =  os_cputime_get32() + ble_ll_rfmgmt_enable_now() + 1;
```

Comment:
why not just make ble_ll_rfmgmt_enable_now() do all those and return exact start time? s